adapters/globalsun/globalsun.go

```diff
@@ -0,0 +1,148 @@
+package globalsun
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type reqBodyExt struct {
+	GlobalsunBidderExt reqBodyExtBidder `json:"bidder"`
```

Comment:
reqBodyExt is not exported so we also do not have to export `GlobalsunBidderExt`. It could just be unexported `globalsunBidderExt`

Reply:
Thank you for writing, I did not quite understand, could you give an example of how to fix this?

Reply:
nvm. this will be required to Unmarshal json into `reqBodyExt` struct. So please ignore this comment.